_posts/2019-07-23-hibernate-orm-config-profiles.adoc

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: Tips to use Hibernate ORM with Quarkus profiles and the dev mode
+date: 2019-07-23 14:00:00.000 -0600
+tags: hibernate
+author: ebernard
+---
+
+= Tips to use Hibernate ORM with Quarkus profiles and the dev mode
+
+Quarkus development mode is really useful for applications that mix front end or services and database access.
+There are a few common approaches to make the best of it.
+
+The first choice is to use `quarkus.hibernate-orm.database.generation=drop-and-create` in conjunction with `import.sql`.
+That way for every change to your app and in particular to your entities, the database schema will be properly recreated
+and your data fixture (stored in `import.sql`) will be used to repopulate it from scratch.
+This is best to perfectly control your environment.
```

Comment:
Should we take this opportunity to highlight that Quarkus is able to live-reload even after changes to the import.sql? Or even clarify that any change to the model is immediately applied to the schema? IMO these are extremely cool, just not sure if it's just me :)